class-tgm-plugin-activation.php

```diff
@@ -926,7 +924,7 @@ public function dismiss() {
 		 * @param array $plugin Array of plugin arguments.
 		 */
 		public function register( $plugin ) {
-			if ( ! isset( $plugin['slug'] ) || ! isset( $plugin['name'] ) ) {
+			if ( empty( $plugin['slug'] ) || empty( $plugin['name'] ) ) {
```

Comment:
`isset()` checks that the variable exists and is not `null`, `empty()` tests whether the variable exists and is not empty for a variety of definitions of "empty". `empty()` will return `true` for non-existent variables without an undefined variable notice (which is why it's often abused by people who don't understand `empty()` properly).

So in this case, I'm enhancing the variable check by testing for existence and a non-empty value.